test/common/user_permissions.go

```diff
@@ -0,0 +1,227 @@
+package common
+
+import (
+	goctx "context"
+	"crypto/tls"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/google/go-querystring/query"
+	"github.com/integr8ly/integreatly-operator/pkg/apis/integreatly/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/test/resources"
+	projectv1 "github.com/openshift/api/project/v1"
+	v1 "github.com/openshift/api/route/v1"
+	corev1 "k8s.io/api/core/v1"
+	"strings"
+
+	"io/ioutil"
+	"k8s.io/apimachinery/pkg/types"
+	"net/http"
+	"testing"
+)
+
+const (
+	openshiftAuthenticationNamespace = "openshift-authentication"
+	openshiftOAuthRouteName          = "oauth-openshift"
+
+	pathProjects = "api/kubernetes/apis/project.openshift.io/v1/projects"
+	pathFusePods = "api/kubernetes/api/v1/namespaces/redhat-rhmi-fuse/pods"
```

Comment:
Does this hard-coded path need to be updated to use the `namespacePrefix` value from the installation CR?